### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by danil-maksimenko**
https://danil-maksimenko.github.io/layout_landing-page/

### CODE REVIEW

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Waiting other sections

Smooth scroll working correctly

chrome-capture-2024-2-11 (2)

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Follow the videos in this and next lessons to implement the page block by block
 - Start writing styles in the `main.scss` file. It is be explained in the `Sass` lesson
 - **DON'T** try to do it `Pixel Perfect` - implement it the most `simple` way so it looks similar;
 - When done check yourself using the [CHECKLIST](./checklist.md) when finished;
-- Deploy and create a Pull Request with a [DEMO LINK](https://<your_account>.github.io/layout_landing-page/)
+- Deploy and create a Pull Request with a [DEMO LINK](https://danil-maksimenko.github.io/layout_landing-page/)
```
Comment (anastasiiavorobiova) [inline README.md:17]: Add this link to the PR description

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Make sure to deploy your page

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Try to redeploy your page 
Screenshot 2024-02-27 at 16 09 27

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! 
Screenshot 2024-02-27 at 16 35 07
Consider adding hover styles here 
Screenshot 2024-02-27 at 16 34 36
Consider showing the phone number on the hover

**Review by vadiimvooo — CHANGES_REQUESTED** [review]
Good job. A few changes required:
1. You need to fix scale of the elements. They should not overlap neighbor elements.
image
2. Remove scroll when burger menu is opened.
image

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just one tiny thing needs to be fixed before approval)

also block must not overlap 
image

**Review by vadiimvooo — APPROVED** [review]
LGTM

**Pull request closed without merging**